tests/src/Simple/MultiModule/no_unix

```diff
@@ -0,0 +1 @@
+Doesn't work on OSX.
```

Comment:
Both Linux and OSX need more finessing with the global tables before they'll link cleanly.